src/Integration.Vsix/Commands/HelpMenu/AboutDialog.xaml

```diff
@@ -0,0 +1,39 @@
+﻿<platformUi:DialogWindow x:Class="SonarLint.VisualStudio.Integration.Vsix.Commands.AboutDialog"
```

Comment:
Please give some input regarding the design and everything, this is still a pretty rough draft